Obsolete_Deprecated/IORefCAS/IORefCAS.cabal

```diff
@@ -61,8 +63,14 @@ Test-Suite test-IORefCAS
     cpp-options: -DT1 -DT2
     if !os(mingw32)
       cpp-options: -DT3
-    build-depends: base >= 4.4 && < 5, time, ghc-prim,
-                   QuickCheck, HUnit, bits-atomic
+    build-depends: base >= 4.4 && < 5
```

Comment:
Now that we're supporting GHC 7.10 as the minimum, we should update all of the lower version bounds on `base` to be `>= 4.9` to reflect this.

Reply:
It should be `>= 4.8`, according to this page: https://downloads.haskell.org/~ghc/7.10.3/docs/html/users_guide/release-7-10-3.html

Reply:
Oops, I was mistaken. Thanks for catching that.